src/Framework/EventGen/XSecAlgorithmI.cxx

```diff
@@ -57,3 +57,13 @@ bool XSecAlgorithmI::ValidKinematics(const Interaction* interaction) const
   return true;
 }
 //___________________________________________________________________________
+const HadronTensorI * XSecAlgorithmI::GetTensor( int tensor_pdg, HadronTensorType_t type ) const {
```

Comment:
These changes have to be removed. We cannot modify the XSecAlgorithmI just to reweight.